central/complianceoperator/v2/report/manager/manager_impl.go

```diff
@@ -0,0 +1,66 @@
+package manager
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/pkg/errors"
+	scanConfigurationDS "github.com/stackrox/rox/central/complianceoperator/v2/scanconfigurations/datastore"
+	"github.com/stackrox/rox/pkg/set"
+)
+
+type managerImpl struct {
+	datastore            scanConfigurationDS.DataStore
+	runningReportConfigs set.StringSet
```

Comment:
a map would be good, scan config id -> report requests

Reply:
We'll need an in mem report request struct to hold some info -> like scan config id, and status. For now we can begin with one report request at a time is processed, and the move on to a bounded number of parallel report requests being processed